evepraisal.toml

```diff
@@ -0,0 +1,58 @@
+#./evepraisal/app.go:279:        if viper.GetString("http_addr") != "" {
```

Comment:
can you delete these comments?

Reply:
Yes; I left them in place, to give users a general idea where to look, to dig further into what the configuration settings do, though.